src/calculateRentalCost.js

```diff
@@ -28,7 +28,9 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
```

Comment:
```suggestion
  const totalCost = days * 40;
```
What is 40? Create variable for this